api.ts

```diff
@@ -36,7 +39,7 @@ export const auth = async ({ host, username, password }: AuthParams) => {
 
     // Make the request to get the auth token
     const response = await axios.post(
-      `${host}/rest/v10/oauth2/token`,
+      `${host}/rest/v${apiVersion}/oauth2/token`,
```

Comment:
refactor all API calls to use a shared constant for the API_URL (host+base api path with version)